Firestore/core/src/firebase/firestore/util/executor_libdispatch.h

```diff
@@ -26,7 +26,7 @@
 #include "dispatch/dispatch.h"
 
 #include "Firestore/core/src/firebase/firestore/util/executor.h"
-#include "Firestore/core/src/firebase/firestore/util/firebase_assert.h"
+#include "Firestore/core/src/firebase/firestore/util/hard_assert.h"
```

Comment:
Is this needed? It doesn't look like the header file uses any of these macros.

Reply:
It wasn't used directly, but was used in the implementation. Moved.